.github/workflows/test.yaml

```diff
@@ -93,6 +94,42 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           files: ./cover.out
-          name: go-livepeer
+          name: ${{ github.event.repository.name }}
           verbose: true
           gcov_ignore: ./eth/stubclient.go
+
+  codeql:
```

Comment:
Is this related to `The dependabot generated PRs don't have access to secrets`?

Reply:
codeql itself isn't related to dependabot.

Reply:
ok, so could you please send it as a separate PR or update the description of this PR?

Reply:
updated the description here.